packages/server-ct/src/routes-ct.ts

```diff
@@ -44,6 +44,18 @@ export function initializeRoutes ({ app, config, project }: InitializeRoutes) {
     // files.handleIframe(req, res, config, getRemoteState, extraOptions)
   })
 
+  app.get(`${config.clientRoute}ctChunk-*`, (req, res) => {
+    debug('Serving Cypress front-end chunk by requested URL:', req.url)
+
+    runnerCt.serveChunk(req, res, { config })
+  })
+
+  app.get(`${config.clientRoute}vendors~ctChunk-*`, (req, res) => {
+    debug('Serving Cypress front-end vendor chunk by requested URL:', req.url)
+
+    runnerCt.serveChunk(req, res, { config })
+  })
```

Comment:
I don't understand why this is necessary...

By hardcoding the chunk names with `vendor` prefix, we will break a lot of webpack configurations, and chunks already working in the existing runner? We should get on the phone and talk about this ☎️

Reply:
This is needed **only for runner** to make possible OUR CODE to be splitteed into chunks with specific naming convention `ctChunk-` and make possible to use dynamic import.

Dynamic import is required to not connect to WebSocket until we get the config and the specified `socketIoRoute` and in the future it will be required to load react devtools only for react